spec/linter-spec.js

```diff
@@ -13,7 +15,9 @@ describe('Js-YAML provider for Linter', () => {
     // Info about this beforeEach() implementation:
     // https://github.com/AtomLinter/Meta/issues/15
     const activationPromise = atom.packages.activatePackage('linter-js-yaml').then(() =>
-      atom.config.set('linter-js-yaml.customTags', ['!yaml', '!include']),
+      atom.config.set('linter-js-yaml.customTags', [
```

Comment:
This works, but it might be clearer to only change this setting for the specs that are testing it.